src/main/java/bio/terra/workspace/app/configuration/external/SpendProfileConfiguration.java

```diff
@@ -0,0 +1,55 @@
+package bio.terra.workspace.app.configuration.external;
+
+import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Configuration for Spend Profiles.
+ *
+ * <p>TODO: Replace the configuration based Spend Profile with integration with the to-be-built
+ * Spend Profile Component. For now, we manage spend profiles manually.
+ */
+@Configuration
+@EnableConfigurationProperties
+@ConfigurationProperties(prefix = "workspace.spend")
+public class SpendProfileConfiguration {
+  /** The Spend Profiles known to Workspace Manager. */
+  private List<SpendProfileModel> spendProfiles = new ArrayList<>();
```

Comment:
I haven't created collections of non-primitive objects via config before - just to confirm, does this pattern work for creating objects?

Reply:
Yep. Yaml for lists & maps works as you want it to. Nested pojo objects are also fine.